consensus/babe/types.go

```diff
@@ -42,3 +42,12 @@ type AuthorityData struct {
 	id     *sr25519.PublicKey
 	weight uint64
 }
+
+// BabeHeader as defined in Polkadot RE Spec, definition 4.10 in section 4.1.4
```

Comment:
definition 5.10 in section 5.1.4

Reply:
It's definition 4.10 in 4.1.4, I had it wrong in the description.